### PR TITLE
DOC: Document installer uninstallation

### LINKED ISSUE #12528: Complete uninstall documentation missing
### Proposed documentation enhancement

For installations on linux using "sh ./MNE-Python-1.6.1_0-Linux.sh", there is no documentation to uninstall the application completely. doing "pip uninstall mne" only removes the package and not the conda environment auto created.

### PR DESCRIPTION
**Pull request by larsoner**
A collaborator recently asked me about uninstallation so I was reminded to do this.

Closes #12528
Closes https://github.com/mne-tools/mne-installers/issues/89

### CODE REVIEW

**Comment by larsoner** [comment]
https://output.circle-artifacts.com/output/job/493e2c81-fdaf-4f23-a57d-a99510976e5e/artifacts/0/html/install/installers.html#installers

(If good enough, should be merged without waiting for reqs because I skipped most CIs)

File: doc/Makefile [hunk 1]
```diff
@@ -76,3 +76,6 @@ view:
 	@python -c "import webbrowser; webbrowser.open_new_tab('file://$(PWD)/_build/html/sg_execution_times.html')"
 
 show: view
+
+serve:
+	python -m http.server -d _build/html
```
Comment (drammock) [inline doc/Makefile:81]: ❤️

File: doc/install/installers.rst [hunk 2]
```diff
@@ -107,3 +110,54 @@ bundles to the ``Applications`` folder on macOS.
...
+
+1. Remove relevant lines from your shell initialization scripts if you
+   added them at installation time. To do this, you can run from the MNE Prompt:
+
+   .. code-block:: bash
+
+       $ conda init --reverse
+
+   Or you can manually edit shell initialization scripts, e.g., ``~/.bashrc`` or
+   ``~/.bash_profile``.
```
Comment (drammock) [inline doc/install/installers.rst:127]: or `.zshrc`? I guess you said "e.g." so not really necessary to name them all

**Review by drammock — APPROVED** [review]

**Comment by drammock** [comment]
thanks @larsoner

**Pull request merged**